.gitlab/kitchen_tests_upload.yml

```diff
@@ -12,39 +12,17 @@ kitchen_tests_upload_common:
     - kitchen_centos_fips_upgrade7_agent-a7
```

Comment:
I think the `centos_fips` dependencies can be removed as well. And you could add the newly created tests to the `e2e_test_junit_upload.yml` file